examples/ds18b20/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"machine"
+	"time"
+	"tinygo.org/x/drivers/onewire"
+
+	"tinygo.org/x/drivers/ds18b20"
+)
+
+func main() {
+	pin := machine.D2
+
+	ow := onewire.New(pin)
+	sensor := ds18b20.New(ow)
+	for {
+		time.Sleep(3 * time.Second)
+
+		println()
+		println("Device:", machine.Device)
+
+		println("Read 1-Wire ROM.")
+		println("Send command =", SliceToHexString([]uint8{onewire.READ_ROM}))
+		err := sensor.ReadAddress()
+		if err != nil {
+			println(err)
+			continue
+		}
+		println("RomID:", SliceToHexString(sensor.RomID))
+
+		println("Request Temperature.")
+		println("Send command =", SliceToHexString([]uint8{ds18b20.DS18B20_CONVERT_TEMPERATURE}))
+		err = sensor.RequestTemperature()
+		if err != nil {
+			println(err)
+			continue
+		}
+
+		// wait 750ms or more for DS18B20 convert T
+		time.Sleep(1 * time.Second)
+
+		println("Read Temperature")
+		println("Send command =", SliceToHexString([]uint8{ds18b20.DS18B20_READ_SCRATCHPAD}))
+		t, err := sensor.ReadTemperature()
+		if err != nil {
+			println(err)
+			continue
+		}
+		// temperature in celsius milli degrees (°C/1000)
+		println("Temperature (°C/1000): ", t)
+	}
+}
+
+// SliceToHexString converts a slice to Hex string
+// fmt.Printf - compile error on an Arduino Uno boards
+func SliceToHexString(rom []uint8) string {
```

Comment:
Just out of curiosity, what was the reason to not use `encoding/hex`?

Reply:
Just forgot about it... ![image](https://user-images.githubusercontent.com/28081613/215255469-46d9d9e9-8f47-4e4c-982f-e63344b46a65.png)